suites/pacific/rados/tier-2_rados_osd_rebalance.yaml

```diff
@@ -0,0 +1,114 @@
+# Suite contains tests related to osd re-balance
```

Comment:
Since we already have general rebalancing tests, I feel this can be updated as ` # Suite contains tests related to osd re-balance upon OSD addition / removal `

Reply:
done